libyear-gradle-plugin/src/main/kotlin/com/libyear/traversal/DependencyTraversal.kt

```diff
@@ -7,7 +7,8 @@ import org.gradle.api.artifacts.result.ResolvedDependencyResult
 
 class DependencyTraversal private constructor(
   private val visitor: DependencyVisitor,
-  private val maxTransitiveDepth: Int?
+  private val maxTransitiveDepth: Int?,
```

Comment:
If we nailed the exclusion list, how hard would it be to add an inclusion list?
Does your list of use-cases contain measuring the libyears of some subset of dependencies?
Do you think that would be useful?

Reply:
My use case is basically that we have a whole namespace which libyear will not be able to access: `com.xyz` so if I just add that string to the exclude list libyear won't even try, which saves time getting 404's from Maven/etc.

Reply:
Oh, I see your example above re: `org.slf4j:slf4j` and `org.slf4j:slf4j-core`
That does make sense. If there is an artifact group we largely want to ignore except for 1 module.

I'll add include!